### PR TITLE
Retries merge

### PR DESCRIPTION
**Pull request by Vassu05**
Retrying the exception and reducing the stack trace.

### CODE REVIEW

File: src/com/oltpbenchmark/api/Worker.java [hunk 1]
```diff
@@ -539,19 +543,20 @@ protected final ArrayList<Pair<TransactionExecutionState, TransactionStatus>> do
...
                 } catch (Error ex) {
-                    LOG.error("Fatal error when invoking " + next, ex);
-                    throw ex;
+                    LOG.error("Fatal error when invoking :" + ex.getMessage());
+                    status = TransactionStatus.RETRY;
                  // Random Error
                 } catch (Exception ex) {
-                    LOG.error("Fatal error when invoking " + next, ex);
-                    throw new RuntimeException(ex);
+                    LOG.error("Fatal error when invoking :" + ex);
```
Comment (sonalsagarwal) [inline src/com/oltpbenchmark/api/Worker.java]: If there are too many exceptions, we will start seeing client side memory issues.

**Review by sonalsagarwal — CHANGES_REQUESTED** [review]

File: src/com/oltpbenchmark/api/Worker.java (hunk 1 above)
Comment (Vassu05) [reply src/com/oltpbenchmark/api/Worker.java]: included stack trace instead of full exception.

File: config/workload_all.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
       Set the number of retries to 0 as retrying when the number of warehouses is
       high is pointless as it just leads to more failures.
     -->
-    <maxRetriesPerTransaction>2</maxRetriesPerTransaction>
+    <maxRetriesPerTransaction>3</maxRetriesPerTransaction>
```
Comment (hbhanawat) [inline config/workload_all.xml]: Please make it 2. Otherwise all the reports will be start showing 3 retries.

Comment (Vassu05) [reply config/workload_all.xml]: Keeping the retries as 2

**Review by sonalsagarwal — APPROVED** [review]

**Pull request merged**